src/lib/services/user-service.test.ts

```diff
@@ -145,6 +145,7 @@ test('Should create default user - with provided username and password', async (
         initialAdminUser: {
             username: 'admin',
             password: 'unleash4all!',
+            email: 'admin@unleash.com',
```

Comment:
should not be set.

Reply:
Assuming we are still keeping the email functionality but making it opt-in, it would be beneficial to have a test for it. Would it be better to create a separate test case for the email rather than having it in this one?